stories/features/welcome/page/index.tsx

```diff
@@ -133,7 +133,7 @@ class WelcomePage extends React.PureComponent<{}, WelcomePageState> {
           </Select>
           <Button
             level='primary'
-            type={this.state.fakeChangedDefaultTheme ? 'default' : 'accent'}
+            type={this.state.fakeChangedDefaultTheme ? 'accent' : 'accent'}
```

Comment:
Could be a possibility to introduce a "success" button style of some sort that incorporates green. We have "warn" for red.

Reply:
I think it's just a matter of setting the right color. we can then open a new issue and address in a follow-up